docs/reference/api/docker_remote_api_v1.24.md

```diff
@@ -3501,6 +3501,89 @@ Return low-level information on the node `id`
 -   **404** – no such node
 -   **500** – server error
 
+### Remove a node
+
+
+`DELETE /nodes/<id>`
+
+Remove a node [`id`] from the Swarm.
```

Comment:
Other descriptions don't seem to use the bracketed format [`id`]

Also, from the code, it looks like DELETE and GET can take either an ID, a full name or an ID prefix, but POST can only use the actual ID. This seems inconsistent (and doesn't match the docs). But I'm not sure whether we should fix the docs or the code.